File: CS/CustomJQuery/Models/Person.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace CustomJQuery.Models
{
    public class Person
    {
        // Fields...
        private bool _CheckAge;
        private DateTime _BirthDate;
        private string _CompanyName;
        private string _Name;
        private int _ID;
        [Required]
        public int ID
        {
            get { return _ID; }
            set { _ID = value; }
        }
        [Required]
        public string Name
        {
            get { return _Name; }
            set { _Name = value; }
        }

        public string CompanyName
        {
            get { return _CompanyName; }
            set { _CompanyName = value; }
        }
        [CustomAttribute(MinDate=20)]
        public DateTime BirthDate
        {
            get { return _BirthDate; }
            set { _BirthDate = value; }
        }

        public bool CheckAge
        {
            get { return _CheckAge; }
            set { _CheckAge = value; }
        }
    }
}

File: VB/CustomJQuery/Models/Person.vb
Imports System.ComponentModel.DataAnnotations

Namespace CustomJQuery.Models

    Public Class Person

        ' Fields...
        Private _CheckAge As Boolean

        Private _BirthDate As Date

        Private _CompanyName As String

        Private _Name As String

        Private _ID As Integer

        <Required>
        Public Property ID As Integer
            Get
                Return _ID
            End Get

            Set(ByVal value As Integer)
                _ID = value
            End Set
        End Property

        <Required>
        Public Property Name As String
            Get
                Return _Name
            End Get

            Set(ByVal value As String)
                _Name = value
            End Set
        End Property

        Public Property CompanyName As String
            Get
                Return _CompanyName
            End Get

            Set(ByVal value As String)
                _CompanyName = value
            End Set
        End Property

        <CustomAttribute(MinDate:=20)>
        Public Property BirthDate As Date
            Get
                Return _BirthDate
            End Get

            Set(ByVal value As Date)
                _BirthDate = value
            End Set
        End Property

        Public Property CheckAge As Boolean
            Get
                Return _CheckAge
            End Get

            Set(ByVal value As Boolean)
                _CheckAge = value
            End Set
        End Property
    End Class
End Namespace
